cypress/integration/resources.spec.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import {
+  RESOURCE_CARD,
+  RESOURCE_SEARCH,
+  RESOURCE_SEARCH_BUTTON,
+  RESOURCE_RESET_BUTTON,
+} from '../../common/constants/testIDs';
+
+describe('resources', () => {
+  const CostSelect = 'input#react-select-paid-input';
+  const LanguageSelect = 'input#react-select-languages-input';
+  const CategorySelect = 'input#react-select-category-input';
+
+  beforeEach(() => {
+    cy.visitAndWaitFor('/resources/1');
+    cy.get('h1').should('have.text', 'Resources');
+  });
+
+  it('redirects on /resources to resources/1', () => {
+    cy.visitAndWaitFor('/resources');
+    cy.location('pathname').should('eq', '/resources/1');
+  });
+
+  it('loads many resource cards on resources/1', () => {
+    cy.visitAndWaitFor('/resources/1');
+    cy.findAllByTestId(RESOURCE_CARD).should('have.length', 20);
+  });
+
+  it('renders the second page of results when using pagination to visit page 2', () => {
+    cy.findByTestId('2').click();
+    cy.location('pathname').should('eq', '/resources/2');
+  });
+
+  it('renders the next page of results when using pagination to visit next page', () => {
+    cy.findByTestId('rightAngle').click();
+    cy.location('pathname').should('eq', '/resources/2');
+  });
+
+  it('renders the previous page of results when using pagination to visit previous page', () => {
+    cy.visitAndWaitFor('/resources/2');
+    cy.findByTestId('leftAngle').click();
+    cy.location('pathname').should('eq', '/resources/1');
+  });
+
+  it('renders the last page of results when paginating to the last page', () => {
+    let pageValue;
+    cy.get('ol')
+      .find('li')
+      .eq(-2)
+      .click()
+      .then($li => {
+        pageValue = $li.text().match(/[0-9]+/);
+      });
+    cy.location().should(loc => {
+      expect(loc.pathname).to.eq(`/resources/${pageValue}`);
+      expect(loc.search).to.eq('');
+    });
+  });
+
+  it('will allow a user to search matching resources by input', () => {
+    cy.findByTestId(RESOURCE_SEARCH).click().type('javascript', { force: true }).type('{enter}');
+
+    cy.findByTestId('2').click();
```

Comment:
Hmmm what has a testid of 2?

Reply:
changed to `page <page number>` to be a more descriptive testid